src/main/java/org/mule/extensions/jms/internal/connection/provider/GenericConnectionProvider.java

```diff
@@ -39,6 +62,11 @@
 @ExternalLib(name = "JMS Client", description = "Client which lets communicate with a JMS broker", type = DEPENDENCY)
 public class GenericConnectionProvider extends BaseConnectionProvider {
 
+  private static final Logger LOGGER = getLogger(GenericConnectionProvider.class);
+  public static final String PROTOCOL = "SSL";
```

Comment:
Why are you adding this constant as public? Where are you using it?

Reply:
This constant was removed, we will use a dynamic protocol assignment